relaxed.module

```diff
@@ -65,7 +65,7 @@ function relaxed_entity_access(EntityInterface $entity, $operation, AccountInter
   $entity_type_id = $entity_type->id();
 
   if ($multiversion_manager->isEnabledEntityType($entity_type) || $entity_type_id == 'replication_log') {
-    return AccessResult::allowedIfHasPermission($account, 'perform content replication');
+    return AccessResult::allowedIfHasPermissions($account, ['perform push replication', 'perform pull replication'], 'OR');
```

Comment:
Don't we need to check what $operation is being triggered here? Pull replication should only be able to read/view and not edit/delete.